Content.Server/Damage/Components/DamageOnHighSpeedImpactComponent.cs

```diff
@@ -8,7 +8,7 @@ namespace Content.Server.Damage.Components;
 /// <summary>
 /// Should the entity take damage / be stunned if colliding at a speed above MinimumSpeed?
 /// </summary>
-[RegisterComponent, Access(typeof(DamageOnHighSpeedImpactSystem))]
+[RegisterComponent]
 public sealed partial class DamageOnHighSpeedImpactComponent : Component
```

Comment:
Please do not remove access modifiers.

Reply:
How do i change these variables without access to the component?

Reply:
Go through a system or add access attributes to specific fields.

Reply:
This is fixed now but reliant on #21227